libr/include/r_util/r_graph.h

```diff
@@ -19,13 +19,13 @@ typedef struct r_graph_node_t {
 typedef struct r_graph_edge_t {
 	RGraphNode *from;
 	RGraphNode *to;
-	int nth;
+	st32 nth;
 } RGraphEdge;
 
 typedef struct r_graph_t {
-	unsigned int n_nodes;
-	unsigned int n_edges;
-	int last_index;
+	ut32 n_nodes;
```

Comment:
What about using size_t instead so we ensure the struct fields are properly aligned in memory without gaps? This change will break abi but we can have the new reordered fields under the newabi ifblock

Reply:
n_nodes is more or less a copy of r_list_length (g->nodes), which is ut32. i see no need to switch to size_t. i'm not sure if size_t has guaranteed minimum size